share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/AfuncW.py

```diff
@@ -0,0 +1,109 @@
+"""

```

Comment:
The name `AfuncW` seems a little cryptic to me (not being a domain expert). Could you think of something more explicit?

Reply:
Another nice comment of mine. The name is understandable enough now. =) (Admittedly, the name still doesn't ring a loud and clear bell with me but that's likely just me not being knowledgeable about the details of the physics here.)